react/components/Payment/ConnectorResponseInfo.tsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types'
+import React, { FunctionComponent } from 'react'
+import { injectIntl, InjectedIntlProps, intlShape } from 'react-intl'
+
+const ConnectorResponseInfo: FunctionComponent<Props & InjectedIntlProps> = ({
+  label,
+  value,
+  intl,
+}) => {
+  return (
+    <div className="more-info__data__item f7">
```

Comment:
This css classes should be added trhough cssModules

Reply:
Yep, I don't think these CSS classes are defined right now